### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by RothAndrew**

### CODE REVIEW

**Review by ac-miller — APPROVED** [review]
LGTM

**Comment by RothAndrew** [comment]
This one needs to wait for an approval from @twofivetechnology

**Review by Xerralt — APPROVED** [review]

**Review by Xerralt — APPROVED** [review]

**Comment by RothAndrew** [comment]
Got approval to use Apache 2.0 from stakeholders via email. Merging.

**Pull request merged**
